grafana-dashboard/overlays/moc-prod/thoth-sli-slo.json

```diff
@@ -94,7 +94,7 @@
       "targets": [
         {
           "exemplar": true,
-          "expr": "up{field=\"user-api-thoth-frontend-prod.apps.smaug.na.operate-first.cloud\", prometheus=\"moc/smaug\"}",
+          "expr": "up{field=\"user-api-thoth-frontend-prod.apps.apps.smaug.na.operate-first.cloud\", cluster=\"$env\"}",
```

Comment:
```suggestion
          "expr": "up{field=\"user-api-thoth-frontend-prod.apps.smaug.na.operate-first.cloud\", cluster=\"$env\"}",
```

Reply:
> lgtm minor changes needed:

It is actually correct, but I suspect there is an issue then: https://github.com/thoth-station/thoth-application/issues/2160